examples/src/keyrings/aws_kms_keyring_example.py

```diff
@@ -0,0 +1,115 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+This example sets up the KMS Keyring
```

Comment:
Can you add some more info?

Is this key expected to be symmetric or asymmetric?

How can I run this example? (i.e. explain the `kms_key_id` arg, point to AWS docs)

What else does this example do? (it sets up the keyring; then what?)

Reply:
A good example intro is in https://github.com/aws/aws-database-encryption-sdk-dynamodb/blob/main/Examples/runtimes/java/DynamoDbEncryption/src/main/java/software/amazon/cryptography/examples/keyring/HierarchicalKeyringExample.java. I would model what you're doing after this.